### PR TITLE
Switch from h4 to h2 in PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by danielbachhuber**
## Changes proposed in this Pull Request:

Switches from h4 to h2 headings in `PULL_REQUEST_TEMPLATE`. This is more semantically correct document structure, and makes it easier to parse the pull request description.

## Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [x] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

N/A

## Testing instructions:

N/A

### CODE REVIEW

**Review by jeherve — CHANGES_REQUESTED** [review]
I'd be okay with that. It does seem a bit big, but I suppose it's just a matter of getting used to it.

Before we merge this, you'd want to update `projects/github-actions/repo-gardening/src/tasks/check-description/index.js` to match, but other than that the change is pretty straightforward

Since we're making changes here, what would you think of an extra level, with other info being an h5 under changes proposed?

**Comment by danielbachhuber** [comment]
> Since we're making changes here, what would you think of an extra level, with other info being an h5 under changes proposed?

@jeherve I'm not sure I follow... can you clarify?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,25 +5,25 @@ Please add testing instructions to projects/plugins/jetpack/to-test.md in a new
 
 Fixes #
 
-#### Changes proposed in this Pull Request:
+## Changes proposed in this Pull Request:
```
Comment (danielbachhuber) [inline .github/PULL_REQUEST_TEMPLATE.md]: @jeherve What do you think about changing this to:

```
## Proposed changes:
```

Comment (jeherve) [reply .github/PULL_REQUEST_TEMPLATE.md]: I like it, short and to the point!

**Comment by jeherve** [comment]
> @jeherve I'm not sure I follow... can you clarify?

Sorry, that wasn't very clear. Something like this:

```diff
diff --git a/.github/PULL_REQUEST_TEMPLATE.md b/.github/PULL_REQUEST_TEMPLATE.md
index eefef8d513..d652e400e1 100644
--- a/.github/PULL_REQUEST_TEMPLATE.md
+++ b/.github/PULL_REQUEST_TEMPLATE.md
@@ -9,7 +9,7 @@ Fixes #
 <!--- Explain what functional changes your PR includes -->
 *
 
-## Other information:
+### Other information
 
 - [ ] Have you written new tests for your changes, if applicable?
 - [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

```

**Comment by danielbachhuber** [comment]
@jeherve Cool, made those changes.

**Review by jeherve — APPROVED** [review]

**Comment by Realtech1** [comment]
! [rejected]//components -> //components  (would clobber existing tag)

**Comment by Realtech1** [comment]
Damarcus Jones Ai Chips

**Comment by Realtech1** [comment]
diff --git a/.github/PULL_REQUEST_TEMPLATE.md b/.github/PULL_REQUEST_TEMPLATE.md
index eefef8d513..d652e400e1 100644
--- a/.github/PULL_REQUEST_TEMPLATE.md
+++ b/.github/PULL_REQUEST_TEMPLATE.md
@@ -9,7 +9,7 @@ Fixes #
 
 *
 
-## Other information:
+### Other information
 
 - [ ] Have you written new tests for your changes, if applicable?
 - [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
 - [ ]

**Pull request merged**